src/stirling/source_connectors/socket_tracer/protocols/mongodb/stitcher_test.cc

```diff
@@ -0,0 +1,368 @@
+/*
+ * Copyright 2018- The Pixie Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "src/stirling/source_connectors/socket_tracer/protocols/mongodb/stitcher.h"
+
+#include <absl/container/flat_hash_map.h>
+
+#include <string>
+#include <utility>
+
+#include "src/common/testing/testing.h"
+
+namespace px {
+namespace stirling {
+namespace protocols {
+namespace mongodb {
+
+using ::testing::IsEmpty;
+using ::testing::SizeIs;
+
+class MongoDBStitchFramesTest : public ::testing::Test {};
+
+Frame CreateMongoDBFrame(uint64_t ts_ns, int32_t request_id, int32_t response_to, bool more_to_come,
+                         std::string doc = "") {
+  mongodb::Frame frame;
+  frame.timestamp_ns = ts_ns;
+  frame.request_id = request_id;
+  frame.response_to = response_to;
+  frame.more_to_come = more_to_come;
+
+  mongodb::Section section;
+  section.documents.push_back(doc);
+
+  frame.sections.push_back(section);
+
+  return frame;
+}
+
+// Uncomment following two functions once upstream testing functions are merged.
+bool areAllDequesEmpty(
```

Comment:
This should be PascalCase per our [style guide ](https://github.com/pixie-io/pixie/blob/e670eb63cb18368d671169fb630567d78cd3f8c8/styleguide/cc_style.md#c-style-guide-proposalsopens) -- `AreAllDequeuesEmpty`. The particular part is in the [Google C++ guide](https://google.github.io/styleguide/cppguide.html#General_Naming_Rules).